### PR TITLE
client/fingerprint: Use GetDiskFreeSpaceEx instead of parsing output of fsutil on Windows.

### PR DESCRIPTION
**Pull request by ChrisHines**
This change avoids a dependency on fsutil, which requires Administrator privileges to execute.

### CODE REVIEW

File: client/fingerprint/storage.go [hunk 1]
```diff
@@ -52,91 +42,17 @@ func (f *StorageFingerprint) Fingerprint(cfg *config.Config, node *structs.Node)
...
-		free, err := strconv.ParseInt(fields[3], 10, 64)
-		if err != nil {
-			return false, fmt.Errorf("Failed to parse storage.bytesfree size in kilobytes")
-		}
-		// Convert from KB to MB
-		node.Resources.DiskMB = int(free / 1024)
-		// Convert from KB to bytes
-		node.Attributes["storage.bytesfree"] = strconv.FormatInt(free*1024, 10)
-	}
+	const mb = 1024 * 1024
```
Comment (dadgar) [inline client/fingerprint/storage.go]: Can we move this to the top of the file and make it `bytesPerMegabyte`

File: client/fingerprint/storage_windows.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	absPath, err := filepath.Abs(path)
+	if err != nil {
+		return "", 0, 0, fmt.Errorf("failed to determine absolute path for %s", path)
+	}
+
+	volume = filepath.VolumeName(absPath)
+
+	absPathp, err := syscall.UTF16PtrFromString(absPath)
+	if err != nil {
+		return "", 0, 0, fmt.Errorf("failed to determine disk space for %s: %v", absPath, err)
```
Comment (dadgar) [inline client/fingerprint/storage_windows.go]: Can you make the error be specific to getting a ptr to that string failing and then move `failed to determine disk space for %s: %v"` to https://github.com/hashicorp/nomad/pull/519/files#diff-f553b17d3b2ea403163ac9f86af7fbb7R45

**Comment by dadgar** [comment]
LGTM. A few nitpicks

**Comment by ChrisHines** [comment]
@dadgar Thanks for the review comments. PTAL

**Comment by cbednarski** [comment]
Thanks @ChrisHines!

**Pull request merged**